verif/tb/ex_sim_tb/Lightweight_ex_sim.sh

```diff
@@ -0,0 +1,11 @@
+CVA6_PATH="/tools/C/eegr-463/al.roberts/2025C3Sim/cva6"
+vcs -full64 -sverilog +lint=TFIPC-L \
+  +incdir+/tools/C/eegr-463/al.roberts/2025C3Sim/cva6/core/include \
+  "${CVA6_PATH}/core/tb/ex_stage_packages.sv" \
+  "${CVA6_PATH}/core/tb/ex_stage_stubs.sv" \
+  "${CVA6_PATH}/core/ex_stage.sv" \
+  "${CVA6_PATH}/core/tb/ex_stage_tb.sv" \
```

Comment:
please update  to use relative paths instead of absolute paths pointing to your local directory.

```suggestion
CVA6_PATH="$PWD/../../../"
vcs -full64 -sverilog +lint=TFIPC-L \
  +incdir+${CVA6_PATH}/core/include \
  "ex_stage_packages.sv" \
  "ex_stage_stubs.sv" \
  "${CVA6_PATH}/core/ex_stage.sv" \
  "ex_stage_tb.sv" \
```